app-multimedia/wiliwili/autobuild/defines

```diff
@@ -0,0 +1,15 @@
+PKGNAME="wiliwili"
+PKGSEC="video"
+PKGDES="Third-party Bilibili client for multiple platforms"
```

Comment:
A desktop client for Bilibili

No need to specify "multiple platforms (cross-platform)" (we don't care, we are using AOSC OS) nor "third-party" (everything is one way or another).